doc/data/messages/s/singledispatch-method/good.py

```diff
@@ -0,0 +1,19 @@
+from functools import singledispatch
+
+
+class GoodBoard:
```

Comment:
```suggestion
class Board:
```
Nitpick : if the differences between good/bad are only problematic code construct vs good code construct it's easier to understand the example